packages/core/test/render3/component_spec.ts

```diff
@@ -152,6 +152,65 @@ describe('component', () => {
 
 });
 
+it('should not invoke renderer destroy method for child views', () => {
```

Comment:
I think this might be confusing as is, since we *do* want to invoke the destroy method for child component views. We just don't want to invoke it for embedded views.

```suggestion
it('should not invoke renderer destroy method for embedded views', () => {
```